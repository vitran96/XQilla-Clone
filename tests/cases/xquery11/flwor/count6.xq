let $a := 5
count $c
return <result a="{$a}" c="{$c}"/>