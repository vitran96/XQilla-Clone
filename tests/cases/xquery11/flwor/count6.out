<result a="5" c="1"/>